junit5-support/src/main/kotlin/in/specmatic/test/SpecmaticJUnitSupport.kt

```diff
@@ -451,13 +453,14 @@ open class SpecmaticJUnitSupport {
 
     private fun getSpecmaticJson(): SpecmaticConfigJson? {
         return try {
-            loadSpecmaticJsonConfig(configFile)
-        }
-        catch (e: ContractException) {
+            if(configFile.endsWith(".$YAML")) {
+                return loadSpecmaticYamlConfig(configFile)
```

Comment:
Can you refactor loadSpecmaticJsonConfig such that it can handle both yaml and json? That way, stub, test and any other command that needs it will be able to use the yaml config. Currently implementation in this branch seems implemented specifically for test.